File: js/1015.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8');
var lines = input.split('\n');
e1 = lines.shift().split(' ')
let x1 = parseFloat(e1[0])
let y1 = parseFloat(e1[1])

e1 = lines.shift().split(' ')
let x2 = parseFloat(e1[0])
let y2 = parseFloat(e1[1])

console.log(Math.pow((Math.pow (x2-x1, 2)) + (Math.pow (y2-y1, 2)), 0.5).toFixed(4))

File: python/PROBLEMS/1015.py
e = input().split(' ')
x1 = float(e[0])
y1 = float(e[1])
e = input().split(' ')
x2 = float(e[0])
y2 = float(e[1])

print('{:.4f}'.format(pow((pow (x2-x1, 2)) + (pow (y2-y1, 2)), 0.5)))
